docs/pages/guide/gridifyGlobalConfiguration.md

```diff
@@ -47,6 +47,13 @@ some ORMs like NHibernate don't support this. You can disable this behavior by s
 - type: `bool`
 - default: `false`
 
+### DefaultStringComparisonIsCaseInsensitive
```

Comment:
if you changed the configuration name, this part should change too. + new section for GridifyMapper configuration